packages/design-system/src/utils/index.js

```diff
@@ -21,5 +21,13 @@ export { default as useLiveRegion } from './useLiveRegion';
 export { default as uniqueEntriesByKey } from './uniqueEntriesByKey';
 export { default as deepMerge } from './deepMerge';
 export { default as useMouseDownOutsideRef } from './useMouseDownOutsideRef';
+export { default as pointerEventsCss } from './pointerEventsCss';
+export { default as objectWithout } from './objectWithout';
+export { default as getKeyboardMovement } from './getKeyboardMovement';
+export { default as areEventsDragging } from './areEventsDragging';
+export { default as usePerformanceTracking } from './usePerformanceTracking';
```

Comment:
Wouldn't this be more suitable in the `tracking` package?

Reply:
Technically yes. Though the tracking package is very lightweight and does not have any external dependencies currently.

Moving this hook would mean adding a dependency on React.

Given that this package is also used outside the story editor (i.e. in the Gutenberg block and the plugin activation notice), we'd need to be careful to not inadvertently increase bundle size for those places.

Reply:
Yes, that was the thought behind not moving it in the tracking package, the tracking package currently doesn't have any dependency.

Reply:
Hm, okay, makes sense. And it needs to be removed from the `story-editor` since now it's used in masks as well, correct?

To me, it would make the most sense to keep it in the editor only and rework the usage of it so that it would be used in the editor only. E.g. when looking at the `performanceTrackingEvents.js`, that's very editor-specific and shouldn't really be in the `tracking` package nor in the `design-system`. So it feels like it's a custom hook made for the editor specifically based on the tracking package.

(It would require a follow-up issue to remove the usage of the hook from separated packages to use it inside the editor only, too much to tackle it in this PR).

Thoughts?

Reply:
If we can keep it solely in the editor and not in other packages, that would be great for sure.

Reply:
hmm, `usePerformanceTracking` is custom hook, if we keep it only in `story-editor`, we cannot even pass it down as a prop to `mask` package 🤔 


https://github.com/GoogleForCreators/web-stories-wp/blob/ac8bd1373a286d36c8d62cf2e2057a19653791a6/packages/masks/src/frame.js#L190-L197

Reply:
I think what Miina is saying is that its usage in the `mask` package needs to be refactored, so that it will only be used in the `story-editor` package in the future; but that this can be done in a follow-up ticket since this PR is already quite big.

Reply:
^ Yep, that's what I was trying to say.

Reply:
@sayedtaqui Could you also perhaps link the follow-up issues to the resolved threads for visibility here and elsewhere? Unless there's a different system for tracking those in case of decoupling. Thoughts?

Reply:
Sure, I am noting down the to-do items, and will create issues at the end and will link in comments.

Reply:
Issue https://github.com/GoogleForCreators/web-stories-wp/issues/10862